microsoft-365/security/office-365-security/fix-error-code-5-7-7xx-in-exchange-online.md

```diff
@@ -22,12 +22,12 @@ This topic describes what you can do if you see status code 550 5.7.7xx in a non
 
 ## 5.7.705: Tenant has exceeded threshold restriction: What you need to know
 
-Internal senders could see this NDR whenever you try to send mail if your tenant was compromised. This usually occus when the majority of traffic from your tenant has been detected as suspicious and has resulted in a ban on sending ability for the tenant. This can also occur if your users send an large amount of bulk mail from Office 365. As stated in the service description, Exchange Online customers who need to send legitimate bulk commercial email (for example, customer newsletters) should use third-party providers that specialize in these services.
-
-Once your users collectively, as a tenant, send a certain amount of suspicious mail through the service, all users can be prevented from sending any mail until the problem is fixed. Users will receive a Non-Delivery Report (NDR) that states:
+Once your users collectively, as a tenant, send a certain amount of suspicious mail through the service, all users can be prevented from sending any mail until the problem is fixed. This is usually a result of compromise (most common) or sending too much bulk mail. Users will receive a Non-Delivery Report (NDR) that states:
 
 - 550 5.7.705 Access denied, tenant has exceeded threshold
 
+In rare cases, this could also happen if you renew your subscription after it has already run out. The reason for this is because it takes time for the service to sync the new subscription information. This should not take more than a day, but the tenant could be blocked from sending email in the meantime. The best way to prevent this is to make sure the subscription does not run out.
```

Comment:
```suggestion
In rare cases, this could also happen if you renew your subscription after it has already run out. The reason for this is that it takes time for the service to sync the new subscription information. This should not take more than a day, but the tenant could be blocked from sending email in the meantime. The best way to prevent this is to make sure the subscription does not run out.
```